cdc/sink/codec/json.go

```diff
@@ -461,6 +461,7 @@ func (d *JSONEventBatchEncoder) AppendRowChangedEvent(e *model.RowChangedEvent)
 		message.Ts = e.CommitTs
 		message.Schema = &e.Table.Schema
 		message.Table = &e.Table.Table
+		message.RowsCount++
```

Comment:
please take a look at `craft` codec, it also batch multiple rows into the same message.